src/transformers/models/dinov2/modeling_dinov2.py

```diff
@@ -729,14 +613,18 @@ def __init__(self, config):
     def get_input_embeddings(self) -> Dinov2PatchEmbeddings:
         return self.embeddings.patch_embeddings
 
+    @check_model_inputs
+    def _forward_with_additional_outputs(
+        self, pixel_values: torch.Tensor, **kwargs: Unpack[TransformersKwargs]
+    ) -> BaseModelOutput:
+        """Additional forward to capture intermediate outputs by `check_model_inputs` decorator"""
+        embedding_output = self.embeddings(pixel_values)
+        output = self.encoder(embedding_output)
+        return output
```

Comment:
This is the most questionable change. For the backbone, we need intermediate hidden states to create `feature_maps` in `forward`, but we capture them outside of the `forward`. Therefore, another method is introduced with the `@check_model_inputs` decorator. This leads to a `torch.export` error